cli/support.go

```diff
@@ -0,0 +1,202 @@
+package cli
+
+import (
+	"archive/zip"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"golang.org/x/xerrors"
+
+	"cdr.dev/slog"
+	"cdr.dev/slog/sloggers/sloghuman"
+	"github.com/coder/coder/v2/cli/clibase"
+	"github.com/coder/coder/v2/coderd/util/tz"
+	"github.com/coder/coder/v2/codersdk"
+	"github.com/coder/coder/v2/support"
+)
+
+func (r *RootCmd) support() *clibase.Cmd {
+	var outputPath string
+	client := new(codersdk.Client)
+	cmd := &clibase.Cmd{
+		Use:   "support <workspace> [<agent>]",
```

Comment:
Should we make this `support workspace <workspace>` in case we want to add more commands in the future?

Reply:
The existing verbiage is based around a "support bundle", so the command shall be `coder support bundle <workspace>`.